public/components/marketing-stats/3.html

```diff
@@ -0,0 +1,38 @@
+<section role="region" aria-label="Statistics" class="px-4 py-16 mx-auto flex flex-col items-center justify-center">
+    
+    <div class="text-center md:w-1/2 lg:w-1/2 w-full mx-auto">
+      <h1 class="font-extrabold text-3xl"> Most Prominent In e-commerce </h1>
+      <p class="mt-4 text-gray-500 mx-auto"> Lorem ipsum is a placeholder text commonly used to demonstrate the visual form of a document or a typeface without relying on meaningful content. 
+      visual form of a document or a typeface without relying on meaningful content.
+      </p>
+    </div>
+    <hr  class="w-full mt-10"/>
+    <div class="flex  gap-x-10 mt-10 flex-wrap items-center justify-center gap-y-10">
+      <div class="text-center bg-green-500 px-8 rounded-md py-2">
+        <h6 class="text-4xl font-bold lg:text-5xl xl:text-6xl text-white">11K+</h6>
+        <p class="text-sm font-medium tracking-widest text-gray-100 uppercase lg:text-base italic">
+          Orders
+        </p>
+      </div>
+      <div class="text-center bg-green-500 px-8 rounded-md py-2">
+        <h6 class="text-4xl font-bold lg:text-5xl xl:text-6xl text-white">10K+</h6>
+        <p class="text-sm font-medium tracking-widest text-gray-100 uppercase lg:text-base italic">
+          Purchased
+        </p>
+      </div>
+      <div class="text-center bg-green-500 px-8 rounded-md py-2">
+        <h6 class="text-4xl font-bold lg:text-5xl xl:text-6xl text-white">40K+</h6>
+        <p class="text-sm font-medium tracking-widest text-gray-100 uppercase lg:text-base italic">
+          Customers
+        </p>
+      </div>
+      <div class="text-center bg-green-500 px-8 rounded-md py-2">
```

Comment:
Not a fan of the `px-8 py-2` spacing, it feels very uneven.

Reply:
Not a fan of `bg-green-500` and `text-white`. A dark background would be preferred.